README.yaml

```diff
@@ -0,0 +1,108 @@
+---
+#
+# This is the canonical configuration for the `README.md`
+# Run `make readme` to rebuild the `README.md`
+#
+
+# Name of this project
+name: charts
+
+# Logo for this project
+#logo: docs/logo.png
+
+# License of this project
+license: "APACHE2"
+
+# Canonical GitHub repo
+github_repo: cloudposse/charts
+
+# Badges to display
+badges:
+  - name: "Build Status"
+    image: "https://travis-ci.org/cloudposse/charts.svg?branch=master"
+    url: "https://travis-ci.org/cloudposse/charts-cloudtrail"
+  - name: "Latest Release"
+    image: "https://img.shields.io/github/release/cloudposse/charts.svg"
+    url: "https://travis-ci.org/cloudposse/charts-cloudtrail/releases"
+  - name: "Slack Community"
+    image: "https://slack.cloudposse.com/badge.svg"
+    url: "https://slack.cloudposse.com"
+
+# Short description of this project
+description: |-
+  This is the Geodesic microservices distribution of Helm charts for Kubernetes.
+
+  Geodesic is the fastest way to get up and running with a rock solid, production grade cloud platform. Use [standalone our shell](https://github.com/cloudposse/geodesic/) to get started quickly and avoid dependency hell. It ships with support for [Kubernetes](https://github.com/kubernetes/kubernetes/), [Kops](https://github.com/kubernetes/kops/), [Helm](https://github.com/kubernetes/helm/) and [Terraform](https://github.com/hashicorp/terraform/).
+
+  What makes this distribution special is we've developed these charts to integrate with third party services like Github for authentication (OAuth2) and Duo for MFA.
+
+  ## How do I enable these repositories?
+
+
+  ### Incubator Repository
+
+  To add the `incubator` chart repository to your local `helm` client, run `helm repo add`:
```

Comment:
Lets move "Incubator repository", "stable respository" to `docs/layout.md` then include that file.